homeassistant/components/synology_dsm/sensor.py

```diff
@@ -208,3 +215,33 @@ def device_info(self) -> Dict[str, any]:
             "sw_version": self._api.information.version_string,
             "via_device": (DOMAIN, self._api.information.serial),
         }
+
+
+class SynoNasSecuritySensor(SynoNasSensor):
+    """Representation a Synology Security sensor."""
+
+    @property
+    def state(self):
+        """Return the state."""
+        return getattr(self._api.security, self.sensor_type)
```

Comment:
The state here is "safe", but maybe I should create a binary sensor of it instead of a sensor.

Like "Synology security safe" : on if safe, off if not, and maybe a details of it in attrs.

(I don't know all possible values)

What do you think @balloob  ?

Reply:
I think you are spot on and it should be a binary sensor.

Reply:
binary sensor yes. There is even a `safety` device class :) 

https://developers.home-assistant.io/docs/entity_binary_sensor#available-device-classes

Note that off means safe.